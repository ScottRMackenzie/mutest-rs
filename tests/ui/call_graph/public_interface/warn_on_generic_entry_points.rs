//@ print-call-graph
//@ print-targets
//@ stdout
//@ stderr
//@ mutest-flags: --crate-kind=mutant-for-external-tests

pub trait Trait {
    fn assoc_fn();
}

pub fn public_generic_fn<T: Trait>() {
    T::assoc_fn()
}
