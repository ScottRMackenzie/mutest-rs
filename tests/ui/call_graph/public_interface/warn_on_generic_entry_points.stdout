
@@@ call graph @@@

entry points:

public public_generic_fn

@@@ targets @@@

targets: 0 total; 0 safe; 0 unsafe (0 tainted)
